portfolio/src/main/webapp/index.html

```diff
@@ -42,9 +42,17 @@ <h1 style=font-size:40px class="heading-font">Apoorva Dhanala's Portfolio</h1>
       <h3 class="links-font"><a href="resume.html">MY RESUME</a></h3>
       <h3 class="links-font"><a href="photography.html">MY PHOTOGRAPHY</a></h3>
       <h3 class="links-font"><a href="favorites.html">FAVORITE SONGS, TV SHOWS, AND MORE!</a></h3>
-      <h3>Click here to see a random quote:</h3>
-      <button onclick="updateGreeting()">Hi!</button>
-      <div id="quote-container"></div>
+    </div>
+    <div class="comments-box">
+      <h2 style=font-size:40px class="heading-font">COMMENTS</h2>
```

Comment:
move the style to a CSS class (or a style on h2)

Also,no need to change it, but you can configure capitalization via CSS too: https://www.w3schools.com/cssref/pr_text_text-transform.asp

Reply:
Just fixed!